### DISCUSSION TITLE
how to package static resource

### DISCUSSION THREAD

**Discussion by sunchuanstc** [eclipse-theia/theia#14658] [Q&A] [answered]
Hi, I have some static resource like *.pdf、*.png  in my-theia-extension/static, how can i package the static resource into theia/examples/browser/lib?

**Comment by msujew** [comment, marked as answer]
Hey @sunchuanstc,

we usually use the [`CopyWebpackPlugin`](https://webpack.js.org/plugins/copy-webpack-plugin/) to copy resources to the destination directory. IIRC it is already used by the generated `gen-webpack.config.js` used by every Theia build. You can extend the `webpack.config.js` to add your own scripts/plugins to the webpack build process.
